testnet/tools/BUILD.bazel

```diff
@@ -1,3 +1,5 @@
+exports_files(["build-guestos-configs.sh"])
+
```

Comment:
Without this, [Build-ic was failing with](https://github.com/dfinity/ic/actions/runs/11281277557/job/31377545054?pr=1970):
`++ bazel query 'rdeps(//..., set(//ic-os/components:hostos-scripts/build-bootstrap-config-image.sh //ic-os/components:hostos-scripts/generate-guestos-config/generate-guestos-config.sh //ic-os/components:ic/generate-replica-config.sh //ic-os/components:ic/ic.json5.template //rs/ic_os/dev_test_tools/launch-single-vm:src/main.rs //rs/tests/driver:src/driver/bootstrap.rs //pre-commit:shfmt-check))'
`

Reply:
Fixed by adding `build-guestos-configs.sh` as a source for the icos_deploy target